functions/utils/index.js

```diff
@@ -0,0 +1,23 @@
+const groupByKey = (key) => (objArr) =>
+  objArr.reduce((acc, curr) => {
+    const type = curr[key];
+    const currAttendeeForType = acc[type] || [];
+
+    return {
+      ...acc,
+      [type]: [...currAttendeeForType, curr],
+    };
+  }, {});
+
```

Comment:
GIVE US SOME TOTALS !!!!!  NICE

Reply:
This is just totals per order.

Reply:
I could possibly setup a hook to request the number of tickets sold.